app/assets/stylesheets/common/base/reviewables.scss

```diff
@@ -0,0 +1,221 @@
+  background-color: $primary-very-low;
+  padding: 0.5em 1em;
+  margin-bottom: 1em;
+  display: flex;
+
+  .all-filters {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+  .reviewable-filter {
+    display: flex;
+    align-items: center;
+    margin-right: 1em;
+
+    label {
+      margin: 0 0.5em 0 0;
+    }
+    input {
+      margin: 0;
+    }
+  }
+
+  .topic-filter {
+    margin-top: 1em;
+    .btn {
+      margin-left: 0.5em;
+    }
+  }
+  .refresh {
+    height: 1em;
+    display: flex;
+  }
+
+  .score-filter {
+    width: 5em;
+  }
+}
+.user-flag-percentage {
+  display: flex;
+  align-items: center;
+  margin-left: 0.5em;
+
+  .percentage-label {
+    margin-right: 0.25em;
+    &.agreed {
+      color: $success;
+    }
+    &.disagreed {
+      color: $danger;
+    }
+    &.ignored {
+      color: $primary-medium;
+    }
+  }
+  .d-icon {
+    font-size: 0.9em;
+  }
+}
+
+.reviewable-item {
+  margin-bottom: 1em;
+  border-bottom: 1px solid dark-light-choose($primary-low, $secondary-low);
+
+  .topic-statuses {
+    float: none;
+  }
+
+  .reviewable-meta-data {
+    color: dark-light-choose($primary-medium, $secondary-medium);
+    display: flex;
+    width: 100%;
+    margin-bottom: 0.5em;
+    font-size: $font-down-1;
+    align-items: center;
+    .reviewable-type {
+      margin-right: 1em;
+    }
+    .created-at {
+      a {
+        color: dark-light-choose($primary-medium, $secondary-medium);
+      }
+      margin-right: 1em;
+    }
+    .score {
+      margin-right: 1em;
+    }
+    .status {
+      color: dark-light-choose($primary-high, $secondary-high);
+    }
+  }
+
+  .reviewable-contents {
+    display: flex;
+  }
+
+  .reviewable-actions {
+    margin-top: 0.5em;
+    display: flex;
+
+    .reviewable-action,
+    .reviewable-action-dropdown {
+      margin-right: 0.5em;
+
+      &.delete-user {
+        @extend .btn-danger;
+      }
+    }
+  }
+  padding-bottom: 1em;
+}
+
+.reviewable-scores {
+  min-width: 50%;
+
+  .user {
+    display: flex;
+    align-items: center;
+
+    .user-flag-percentage {
+      margin-left: 0.5em;
+    }
+  }
+  th {
+    text-align: left;
+  }
+  th,
+  td {
+    padding: 0.25em;
+  }
+}
+
+.reviewable-conversation {
+  margin: 0.5em 0;
+
+  .reviewable-conversation-post {
+    max-width: $topic-body-width;
+    margin-bottom: 0.5em;
+
+    .username {
+      font-weight: bold;
+      margin-right: 0.25em;
+    }
+  }
+
+  .controls {
+    margin-top: 0.25em;
+  }
+}
+
+.reviewable-queued-post,
+.reviewable-flagged-post {
+  .reviewable-contents {
+    margin-top: 1em;
+  }
+
+  .post-title {
+    background-color: yellow;
+  }
+  .created-by {
+    margin-right: 1em;
+  }
+
+  .post-contents {
+    width: 100%;
+  }
+  .post-topic {
+    font-weight: bold;
+    color: $primary-medium;
+    margin-bottom: 1em;
+  }
+}
+
+.reviewable-flagged-post {
+  .post-body {
+    max-width: $topic-body-width;
+    max-height: 300px;
+    overflow-y: auto;
+
+    p,
```

Comment:
html styling

Reply:
I have no other control over this formatting since it's the `cooked` of a post.